northstar/src/runtime/state.rs

```diff
@@ -766,10 +766,12 @@ impl<'a, L: Launcher> State<'a, L> {
     }
 
     async fn notification(&self, n: Notification) {
-        self.events_tx
-            .send(Event::Notification(n))
-            .await
-            .expect("Internal channel error on main");
+        if !self.events_tx.is_closed() {
+            self.events_tx
+                .send(Event::Notification(n))
+                .await
+                .expect("Internal channel error on main");
+        }
```

Comment:
why not just ignore the send error...this seems racy and all the send error indicates is a closed or dropped channel anyway